### ISSUE TITLE
508: Programmatic table names are generic

### ISSUE THREAD

**Issue by vishnunavuluri** [US-EPA-CAMD/PI-1#2400]
Issue: Data tables contain captions or aria labels, but they are generic and or duplicated.

In the facilities tab, the data tables have a generic aria-label. They should be labeled by the preceding text
role="table" aria-label="Data Table"

Example: Any data tables on the Configurations tab. Systems, Location Attributes etc.

**Comment by annalbrecht** [comment]
Closed ticket based on discussion with CAMD on 3/4/24. Before deployment there will be a scan performed and any issues will be remediated at that time.
